components/abstractions/src/main/java/com/microsoft/kiota/ApiExceptionBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.microsoft.kiota;
+
+import jakarta.annotation.Nonnull;
+
+import java.util.Objects;
+
+/** Builder class for ApiException. */
+public class ApiExceptionBuilder {
+
+    private ApiException value = null;
+
+    /**
+     * Constructs an empty ApiExceptionBuilder
+     */
+    public ApiExceptionBuilder() {
+    }
+
+    /**
+     * Constructs an ApiExceptionBuilder starting from a base ApiException
+     */
+    public ApiExceptionBuilder(ApiException base) {
+        value = base;
+    }
+
+    /**
+     * Assign the message to the builder
+     * @return The builder object.
+     */
+    public ApiExceptionBuilder withMessage(@Nonnull String message) {
```

Comment:
missing doc comment for the parameter (for all methods)

Reply:
done, hopefully 🤞